app/src/main/java/amazingme/activities/app/ForgotPasswordActivity.java

```diff
@@ -0,0 +1,40 @@
+package amazingme.activities.app;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import com.amazingme.activities.R;
+
+public class ForgotPasswordActivity extends AppCompatActivity {
+
+    private Button backFPBtn, sendPBtn;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_forgot_password);
+
+        backFPBtn = (Button) findViewById(R.id.backFPBtn);
+        sendPBtn = (Button) findViewById(R.id.sendPBtn);
```

Comment:
can we pretty please get rid of the "FP" part? 

it's enclosing class should be enough context

Reply:
We already have a backBtn already used as a reference for another back button. That's why I named this one backFPBtn which stands for back button for forgot password. But you simply want to change the variable name so that it's backBtn = (Button) findViewById(R.id.backFPBtn)?

Reply:
yes. 

where is the backBtn code? if it's in another class, then the class name should give us enough context to the page and functionality. if it's in this class, we need a more verbose name to say specifically what it's doing.

Reply:
fixed, along with sendPBtn to sendBtn